packages/backend/src/peripherals/database/TokenRepository.ts

```diff
@@ -0,0 +1,71 @@
+import { SpotAssetId } from '@explorer/types'
+import { TokenRow } from 'knex/types/tables'
+
+import { Logger } from '../../tools/Logger'
+import { toSerializableJson } from '../../utils/toSerializableJson'
+import { BaseRepository } from './shared/BaseRepository'
+import { Database } from './shared/Database'
+
+export interface TokenRecord {
+  assetTypeHash: string
+  assetHash: SpotAssetId
+  tokenId: string | null
+  uri: string | null
+  contractError: unknown[]
+}
+
+export class TokenRepository extends BaseRepository {
+  constructor(database: Database, logger: Logger) {
+    super(database, logger)
+
+    /* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
reenable at the end

Reply:
I think all the other repositories also disable this rule. Do we also want to reenable those?